### PR TITLE
Support upload files

### PR DESCRIPTION
**Pull request by Nyholm**
This will add continue the work in #24

### CODE REVIEW

File: src/Driver/Userland/ConnectionHandler/ConnectionHandler.php [hunk 1]
```diff
@@ -400,6 +400,7 @@ private function dispatchRequest($requestId)
         }
 
         $this->endRequest($requestId);
+        $request->cleanUploadedFiles();
```
Comment (AndrewCarterUK) [inline src/Driver/Userland/ConnectionHandler/ConnectionHandler.php]: Should we maybe put this in a:

```php
try {
    // ...
} finally {
    $request->cleanUploadedFiles();
}
```

**Comment by Nyholm** [comment]
Great suggestions. Thank you for the review. 

I've updated the PRs accordingly.

**Comment by AndrewCarterUK** [comment]
Great work, thanks @Nyholm and @khanhicetea !

**Comment by khanhicetea** [comment]
Thanks @Nyholm , you made my day ! 👍

**Pull request merged**